src/CommunityToolkit/Xamarin.CommunityToolkit/Views/TabView/TabView.shared.cs

```diff
@@ -991,9 +1007,21 @@ void UpdateTabIndicatorPlacement(TabIndicatorPlacement tabIndicatorPlacement)
 			UpdateTabIndicatorMargin();
 		}
 
-		void UpdateIsSwipeEnabled(bool isSwipeEnabled) => contentContainer.IsSwipeEnabled = isSwipeEnabled;
```

Comment:
`contentContainer` is guaranteed not to be null here since it's readonly and set in the constructor (line 103).
Same comment for other `contentContainer` check bellow.